docs/advanced.rst

```diff
@@ -150,6 +150,18 @@ To do this, specify the ``--intermediate`` argument during bundle creation::
 Note that you can specify the ``--intermediate`` argument multiple times to
 include multiple intermediate certificates to your bundle signature.
 
+Passphrase Handling
+~~~~~~~~~~~~~~~~~~~
+
+If the signing key is protected by a passphrase it has to entered
+during signing operation. For automation purpose the passphrase can be
+set in the environment variable RAUC_KEY_PASSPHRASE.
+
+.. note::
+  Since the environment of other processes is visible on certain
+  platforms (e.g. ps under certain Unix OSes) this option should be
+  used with caution.
```

Comment:
We only support Linux.

Reply:
That was copied from the openssl man page :) fixed